Sources/edge/cli/commands/RunCommand.swift

```diff
@@ -241,6 +245,8 @@ struct RunCommand: AsyncParsableCommand, Sendable {
                     } else {
                         $0.cmd = "/bin/\(imageName)"
                     }
+                    $0.appConfig = appConfigData
+                    $0.autoRestart = !debug
```

Comment:
is this used anywhere?

Reply:
Not yet, I should't have pushed this yet. It's for a next PR